[0m19:35:26.100192 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E6675C60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E75228C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E75226E0>]}


============================== 19:35:26.100192 | b32e9617-5b35-400b-8051-89f0a541def7 ==============================
[0m19:35:26.100192 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:35:26.108708 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\huynh\\.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m19:35:42.950859 [debug] [MainThread]: Starter project path: c:\Users\huynh\Desktop\DataWarehouse\.venv\lib\site-packages\dbt\include\starter_project
[0m19:35:43.062602 [info ] [MainThread]: 
Your new dbt project "dw_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m19:35:43.062602 [info ] [MainThread]: Setting up your profile.
[0m19:37:45.932587 [info ] [MainThread]: Profile dw_project written to C:\Users\huynh\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m19:37:45.935380 [debug] [MainThread]: Command `dbt init` succeeded at 19:37:45.935380 after 139.92 seconds
[0m19:37:45.936743 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E6675C60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E7522680>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F1E7522F80>]}
[0m19:37:45.937762 [debug] [MainThread]: Flushing usage events
[0m19:37:47.103790 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:37:58.685180 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022935B85C60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022936A328C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022936A326E0>]}


============================== 19:37:58.690356 | 79be5ac1-0e78-43a2-841a-16af738bf795 ==============================
[0m19:37:58.690356 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:37:58.690356 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\huynh\\.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m19:37:58.728873 [info ] [MainThread]: dbt version: 1.9.4
[0m19:37:58.728873 [info ] [MainThread]: python version: 3.10.11
[0m19:37:58.728873 [info ] [MainThread]: python path: c:\Users\huynh\Desktop\DataWarehouse\.venv\Scripts\python.exe
[0m19:37:58.728873 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m19:37:59.265851 [info ] [MainThread]: Using profiles dir at C:\Users\huynh\.dbt
[0m19:37:59.265851 [info ] [MainThread]: Using profiles.yml file at C:\Users\huynh\.dbt\profiles.yml
[0m19:37:59.265851 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\huynh\Desktop\DataWarehouse\dbt\dbt_project.yml
[0m19:37:59.265851 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m19:37:59.272999 [debug] [MainThread]: Command `dbt debug` failed at 19:37:59.272999 after 0.67 seconds
[0m19:37:59.272999 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022935B85C60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022937535DB0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022937537640>]}
[0m19:37:59.272999 [debug] [MainThread]: Flushing usage events
[0m19:38:00.444298 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:41:09.356350 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05867DCC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05A532950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05A532770>]}


============================== 19:41:09.356350 | f5afd442-cb7a-43f1-9d08-0a16a4baac5d ==============================
[0m19:41:09.356350 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:41:09.356350 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\huynh\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m19:41:09.398266 [info ] [MainThread]: dbt version: 1.9.4
[0m19:41:09.398266 [info ] [MainThread]: python version: 3.10.11
[0m19:41:09.398266 [info ] [MainThread]: python path: c:\Users\huynh\Desktop\DataWarehouse\.venv\Scripts\python.exe
[0m19:41:09.398266 [info ] [MainThread]: os info: Windows-10-10.0.26100-SP0
[0m19:41:10.008154 [info ] [MainThread]: Using profiles dir at C:\Users\huynh\.dbt
[0m19:41:10.009736 [info ] [MainThread]: Using profiles.yml file at C:\Users\huynh\.dbt\profiles.yml
[0m19:41:10.009736 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\huynh\Desktop\DataWarehouse\dbt\dbt_project.yml
[0m19:41:10.009736 [info ] [MainThread]: adapter type: snowflake
[0m19:41:10.009736 [info ] [MainThread]: adapter version: 1.9.4
[0m19:41:10.009736 [info ] [MainThread]: Configuration:
[0m19:41:10.016281 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m19:41:10.017732 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m19:41:10.019017 [info ] [MainThread]: Required dependencies:
[0m19:41:10.020124 [debug] [MainThread]: Executing "git --help"
[0m19:41:10.065952 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m19:41:10.065952 [debug] [MainThread]: STDERR: "b''"
[0m19:41:10.065952 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m19:41:10.065952 [info ] [MainThread]: Connection:
[0m19:41:10.065952 [info ] [MainThread]:   account: BZMMWMW-OV26861
[0m19:41:10.065952 [info ] [MainThread]:   user: vanthanhhuynh
[0m19:41:10.074444 [info ] [MainThread]:   database: EDW
[0m19:41:10.074444 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m19:41:10.076684 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m19:41:10.076684 [info ] [MainThread]:   schema: Silver
[0m19:41:10.079190 [info ] [MainThread]:   authenticator: None
[0m19:41:10.080241 [info ] [MainThread]:   oauth_client_id: None
[0m19:41:10.081872 [info ] [MainThread]:   query_tag: None
[0m19:41:10.082902 [info ] [MainThread]:   client_session_keep_alive: False
[0m19:41:10.083452 [info ] [MainThread]:   host: None
[0m19:41:10.084588 [info ] [MainThread]:   port: None
[0m19:41:10.085563 [info ] [MainThread]:   proxy_host: None
[0m19:41:10.086563 [info ] [MainThread]:   proxy_port: None
[0m19:41:10.088485 [info ] [MainThread]:   protocol: None
[0m19:41:10.088831 [info ] [MainThread]:   connect_retries: 1
[0m19:41:10.090038 [info ] [MainThread]:   connect_timeout: None
[0m19:41:10.091033 [info ] [MainThread]:   retry_on_database_errors: False
[0m19:41:10.092104 [info ] [MainThread]:   retry_all: False
[0m19:41:10.093160 [info ] [MainThread]:   insecure_mode: False
[0m19:41:10.094230 [info ] [MainThread]:   reuse_connections: True
[0m19:41:10.095327 [info ] [MainThread]: Registered adapter: snowflake=1.9.4
[0m19:41:10.528013 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m19:41:12.732161 [debug] [MainThread]: Using snowflake connection "debug"
[0m19:41:12.733186 [debug] [MainThread]: On debug: select 1 as id
[0m19:41:12.733186 [debug] [MainThread]: Opening a new connection, currently in state init
[0m19:41:14.547896 [debug] [MainThread]: SQL status: SUCCESS 1 in 1.815 seconds
[0m19:41:14.550069 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m19:41:14.551010 [info ] [MainThread]: [31m1 check failed:[0m
[0m19:41:14.551895 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\huynh\Desktop\DataWarehouse\dbt\dbt_project.yml> not found

[0m19:41:14.552968 [debug] [MainThread]: Command `dbt debug` failed at 19:41:14.552968 after 5.29 seconds
[0m19:41:14.554101 [debug] [MainThread]: Connection 'debug' was left open.
[0m19:41:14.554101 [debug] [MainThread]: On debug: Close
[0m19:41:14.720777 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05867DCC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05A503760>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001C05A98DF00>]}
[0m19:41:14.722675 [debug] [MainThread]: Flushing usage events
[0m19:41:15.826583 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m14:10:05.266513 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795634DC00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795858CA30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795858C880>]}


============================== 14:10:05.271848 | 5a170c7d-bba8-4fbe-8f46-1049b16e8c18 ==============================
[0m14:10:05.271848 [info ] [MainThread]: Running with dbt=1.5.11
[0m14:10:05.274726 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\huynh\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m14:10:05.274726 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\huynh\Desktop\DataWarehouse\dbt\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:10:05.274726 [debug] [MainThread]: Command `dbt run` failed at 14:10:05.274726 after 0.03 seconds
[0m14:10:05.274726 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795634DC00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795858C880>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002795858CB20>]}
[0m14:10:05.274726 [debug] [MainThread]: Flushing usage events
